docs/examples/device-flow.otg.yaml

```diff
@@ -0,0 +1,75 @@
+devices:
+- ethernets:
+  - ipv4_addresses:
+    - address: 192.0.2.1
+      gateway: 192.0.2.2
+      name: otg1.eth[0].ipv4[0]
+      prefix: 24
+    mac: 02:00:00:00:01:aa
+    mtu: 1500
+    name: otg1.eth[0]
+    port_name: p1
+  name: otg1
```

Comment:
Why are you naming devices `otg1`, `otg2`, etc?  Your flows are named `f1`, `f2`, etc.?  Perhaps devices should be named `d1`, `d2` (or `dev1`, `dev2`)? Minor nit-pick.

Reply:
There might be other devices in the emulated network called like that. No way someone calls something `otg` outside of OTG config.